model_utils/tracker.py

```diff
@@ -79,16 +80,34 @@ def __get__(field, instance, owner):
                     self.saved_data[field.field_name] = deepcopy(value)
                 return data[field.field_name]
 
+        class FileDescriptorTracker(FileDescriptor):
```

Comment:
Is it possible to integrate this with `DeferredAttributeTracker` in a way that doesn't duplicate so much code between them?